react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      startTime: 0,
+      isPlaying: false,
+      isPaused: true
```

Comment:
You can remove one of these. If `isPlaying` is `false`, then that implies that `isPaused` is `true`.